etc/crontab.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
```

Comment:
![cron_groups](https://user-images.githubusercontent.com/38316221/72780928-bfa76b00-3c28-11ea-9303-f72bead37ebe.png)
I recommends to create a Bolt group, in which a Bolt team or merchant can setup time to run jobs regarding their needs.